lib/promscrape/scrapework.go

```diff
@@ -877,7 +904,7 @@ func (sw *scrapeWork) addAutoTimeseries(wc *writeRequestCtx, name string, value
 	sw.addRowToTimeseries(wc, &sw.tmpRow, timestamp, false)
```

Comment:
As far as I see, this method may fail as well and there is no check for it.

Reply:
this method will work as it used to work before, as it has needRelabel arg set to false